src/client/java/dev/creesch/model/WebsocketMessageBuilder.java

```diff
@@ -77,10 +87,14 @@ private static boolean isPing(Text message, MinecraftClient client) {
                 return true;
             }
 
-            String displayName = client.player.getDisplayName().getString();
-            if (pingPattern(displayName).matcher(messageString).find()) {
-                return true;
+            if (client.player.getDisplayName() != null) {
+                String displayName = client.player.getDisplayName().getString();
+                if (pingPattern(displayName).matcher(messageString).find()) {
+                    return true;
+                }
             }
+
+
```

Comment:
Why the extra newlines?

Reply:
I like extra lines? Probably tried something there at some point, removed it and didn't notice the new lines.